src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -410,10 +403,37 @@ public void actionPerformed(final ActionEvent e) {
     });
     menuGame.add(mapZoom).setMnemonic(KeyEvent.VK_Z);
   }
+  
+  private void addUnitNationDrawMenu(final JMenu parentMenu){
+    final JMenu unitSizeMenu = new JMenu();
+    unitSizeMenu.setMnemonic(KeyEvent.VK_N);
+    unitSizeMenu.setText("Show Unit Nation");
+    final ButtonGroup unitNationGroup = new ButtonGroup();
+    final JRadioButtonMenuItem nationOption0 = new JRadioButtonMenuItem("Hide Nation");
```

Comment:
Nit:  vertical distance could be minimized. IE: avoid declaring variables at top, declare them at the last possible moment so they are as close to the usage as you can.